dev-infra/caretaker/check/github.ts

```diff
@@ -37,14 +47,32 @@ async function getGithubInfo(git: GitClient, {githubQueries: queries = []}: Care
     graphQlQuery[queryKey] = params(
         {
           type: 'ISSUE',
+          first: 100,
           query: `"${repoFilter} ${query.replace(/"/g, '\\"')}"`,
         },
-        {issueCount: types.number},
+        {
+          issueCount: types.number,
+          edges: [{
+            node: {...onUnion({
+              PullRequest: {
+                title: types.string,
+                url: types.string,
+              },
+              Issue: {
+                title: types.string,
+                url: types.string,
+              }
+            })}
+          }]
+        },
```

Comment:
You actually don't need to use edges here.
```suggestion
        {
          issueCount: types.number,
          nodes: [{...onUnion({
              PullRequest: {
                title: types.string,
                url: types.string,
              },
              Issue: {
                title: types.string,
                url: types.string,
              }
            })
          }]
        },
```